incubator/buzzfeed-sso/templates/proxy-deployment.yaml

```diff
@@ -0,0 +1,108 @@
+{{- $fullName := include "buzzfeed-sso.fullname" . -}}
+{{- $proxySecret := .Values.proxy.customSecret | default ($fullName) -}}
+{{- $name := include "buzzfeed-sso.name" . -}}
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: {{ $fullName }}-proxy
+  labels:
+    app: {{ $name }}
+    chart: {{ template "buzzfeed-sso.chart" . }}
+    component: {{ $name }}-proxy
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.proxy.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ $name }}
+      component: {{ $name }}-proxy
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
+      labels:
+        app: {{ $name }}
+        component: {{ $name }}-proxy
+        release: {{ .Release.Name }}
+    spec:
+      volumes:
+        - name: {{ $fullName }}
+          configMap:
+            name: {{ $fullName }}
+      containers:
+        - name: {{ .Chart.Name }}-proxy
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command: ["/bin/sso-proxy"]
+          ports:
+            - name: http
+              containerPort: 8080
+              protocol: TCP
+          env:
+            - name: CLIENT_ID
+              valueFrom:
+                secretKeyRef:
+                  name: {{ $proxySecret }}
+                  key: proxy-client-id
+            - name: CLIENT_SECRET
+              valueFrom:
+                secretKeyRef:
+                  name: {{ $proxySecret }}
+                  key: proxy-client-secret
+            - name: COOKIE_SECRET
+              valueFrom:
+                secretKeyRef:
+                  name: {{ $proxySecret }}
+                  key: proxy-cookie-secret
+            - name: EMAIL_DOMAIN
+              value: {{ .Values.emailDomain | quote }}
+            - name: UPSTREAM_CONFIGS
+              value: /sso/upstream_configs.yml
+            - name: PROVIDER_URL
+              value: https://{{ .Values.auth.domain }}
+            # STATSD_HOST and STATSD_PORT must be defined or the app wont launch, they dont need to be a real host / port, but they do need to be defined.
+            - name: STATSD_HOST
+              value: localhost
+            - name: STATSD_PORT
+              value: "11111"
+            - name: COOKIE_SECURE
+              value: "true"
+            - name: CLUSTER
+              value: {{ .Values.proxy.cluster | quote }}
+            - name: VIRTUAL_HOST
+              value: {{ .Values.proxy.virtualHost | quote }}
+          {{- if .Values.proxy.providerUrlInternal }}
+            - name: PROXY_PROVIDER_URL
```

Comment:
@darioblanco this has been renamed PROVIDER_URL_INTERNAL apparently (see https://github.com/buzzfeed/sso/releases/tag/v1.1.0)

Reply:
Ouch, good catch